packages/neon_framework/lib/src/pages/home.dart

```diff
@@ -105,73 +105,71 @@ class _HomePageState extends State<HomePage> {
 
   @override
   Widget build(BuildContext context) {
-    const drawer = NeonDrawer();
-    const appBar = NeonAppBar();
-
-    final appView = ResultBuilder.behaviorSubject(
+    return ResultBuilder.behaviorSubject(
       subject: _appsBloc.appImplementations,
       builder: (context, appImplementations) {
+        Widget appView;
         if (!appImplementations.hasData) {
-          return const SizedBox();
-        }
-
-        if (appImplementations.requireData.isEmpty) {
-          return Center(
+          appView = const SizedBox();
+        } else if (appImplementations.requireData.isEmpty) {
+          appView = Center(
             child: Text(
               NeonLocalizations.of(context).errorNoCompatibleNextcloudAppsFound,
               textAlign: TextAlign.center,
             ),
           );
+        } else {
+          appView = StreamBuilder(
+            stream: _appsBloc.activeApp,
+            builder: (context, activeAppSnapshot) {
+              if (!activeAppSnapshot.hasData) {
+                return const SizedBox();
+              }
+
+              return SafeArea(
+                child: activeAppSnapshot.requireData.page,
+              );
+            },
+          );
         }
 
-        return StreamBuilder(
-          stream: _appsBloc.activeApp,
-          builder: (context, activeAppSnapshot) {
-            if (!activeAppSnapshot.hasData) {
-              return const SizedBox();
-            }
+        final showDrawer = !appImplementations.hasData || appImplementations.requireData.length > 2;
```

Comment:
Because we don't know how many apps will be there. To me it makes more sense to show the drawer in that case, but we could also hide it. This situation only happens on the very first start when no cached data is available.

Reply:
Let's hide it.
i'd rather show it delayed than having it flicker.
And on lower ene devices where the cache is slow the initial load can take some time